usaspending_api/download/migrations/0004_download_views.py

```diff
@@ -54,16 +54,4 @@ class Migration(migrations.Migration):
             sql=[f"{Path('usaspending_api/download/sql/vw_financial_accounts_by_program_activity_object_class_download.sql').read_text()}"],
```

Comment:
Do the new dependencies added by the previous PR need to stay? (I couldn't comment on unchanged lines)

Reply:
Can get that updated as well. Doesn't case any harm to leave it, but should probably remove it since the dependencies aren't actually needed.